docs/bugs_and_glitches.md

```diff
@@ -68,6 +68,7 @@ Some fixes are mentioned as breaking compatibility with link battles. This can b
 - [Swimming NPCs aren't limited by their movement radius](#swimming-npcs-arent-limited-by-their-movement-radius)
 - [`CheckOwnMon` only checks the first five letters of OT names](#checkownmon-only-checks-the-first-five-letters-of-ot-names)
 - [Catching a Transformed Pokémon always catches a Ditto](#catching-a-transformed-pokémon-always-catches-a-ditto)
+- [If your party and current PC box are full during the Dude's catching tutorial, his Poké Ball can't be used and may crash the game](#if-your-party-and-current-pc-box-are-full-during-the-dudes-catching-tutorial-his-poké-ball-can't-be-used-and-may-crash-the-game)
```

Comment:
```suggestion
- [If your party and current PC box are full during the Dude's catching tutorial, his Poké Ball can't be used and may crash the game](#if-your-party-and-current-pc-box-are-full-during-the-dudes-catching-tutorial-his-pok%C3%A9-ball-cant-be-used-and-may-crash-the-game)
```

Reply:
There's no need to %-encode the "é", just remove the ' from "can't".

Reply:
Fixed